### ISSUE TITLE
Vars file

### ISSUE THREAD

**Issue by churnd** [hashicorp/tfc-workflows-github#10]
Is there a way to specify a vars file to use?  I see it is possible to specify vars individually via env vars but on larger projects, that's not exactly sustainable.  I guess the equivalent of `terraform apply -var-file="testing.tfvars"`?

**Comment by srlynch1** [comment]
This is achievable today the way you can approach this is to just output a tfvars file into the checked out repo prior to configuration upload

**Comment by sany2k8** [comment]
@srlynch1 how to output an e.g. `env/config.tfvars` file into the checked-out repo before configuration upload?

**Comment by churnd** [comment]
You add a step to your pipeline to copy the config one directory up when the pipeline executes.

**Comment by sany2k8** [comment]
@churnd Yes I am planning to do something like this in my workflow piple step, but I was expecting to pass some params instead of doing this,

```
      - name: Generate environment wise auto.tfvars
        run: |
          infra_env=$INPUT_ENVIRONMENT
          cp env/${infra_env}.tfvars ${infra_env}.auto.tfvars
        env:
          INPUT_ENVIRONMENT: ${{ inputs.environment }}
```

**Comment by varnav** [comment]
Can be made simpler:

```yaml
     - name: Generate environment wise auto.tfvars
       run: cp ${{ inputs.environment }}.tfvars ${{ inputs.environment }}.auto.tfvars
```
